.github/workflows/_static-analysis.yml

```diff
@@ -67,34 +67,22 @@ jobs:
     runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
-      - uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
-        with:
-          python-version: "3.11"
+      - uses: jdx/mise-action@146a28175021df8ca24f8ee1828cc2a60f980bd5 # v3.5.1
       - uses: actions/cache/restore@0057852bfaa89a56745cba8c7296529d2fc39830 # v4.3.0
-        name: Restore Python Cache
-        id: cache
+        name: Restore pip cache
+        id: pip-cache
         with:
           path: ~/.cache/pip
           key: ubuntu-24.04-${{ runner.arch }}-pip-${{ hashFiles('.github/requirements.txt') }}
-      - name: Install Python Dependencies
-        run: |
-          pip install -r .github/requirements.txt
       - uses: ./.github/actions/setup-node
```

Comment:
Wouldn't the mise-action step above this allow us to remove this step as well?

Reply:
I think so yes.